XestiMonitors.xcodeproj/project.pbxproj

```diff
@@ -25,299 +25,303 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
-		29131075DA99AE9E5B7182D08A196B13 /* AccelerometerMonitor.swift in Sources */ = {isa = PBXBuildFile; fileRef = E7E79CF9A86E720EF1CCC9C293B47BF9 /* AccelerometerMonitor.swift */; };
```

Comment:
Please make sure you have `xUnique` installed, and then run `./Scripts/uniquify_projects`.